js/app/loader.js

```diff
@@ -7,22 +8,21 @@ var loaderGFW = {
   },
 
   urls_dev: {
-    header: 'http://localhost:8000/src/header.html',
-    footer: 'http://localhost:8000/src/footer.html',
-    css: 'http://localhost:8000/css/build/global.css',
+    header: 'http://localhost:9000/src/header.html',
```

Comment:
I changed this because port 8000 conflicts with the GFW API, if you're running both at the same time.